frontend/src/components/RegistrationView/index.tsx

```diff
@@ -1,12 +1,28 @@
 import React, { useState, useEffect } from 'react';
 import RegistrationForm from './Form';
 import { Wrapper, Header, Confirmation, SuccessIcon, Error, Button } from './StyledComponents';
-import { get } from '../../apis';
+import { get, post } from '../../apis';
 
 const RegistrationView: React.FC = (props) => {  
     const  [submitted, setSubmitted] = useState(false);
     const [clubs, setClubs] = useState([]);
     const [error, setError] = useState(false);
+    const [auth, setAuth] = useState(false);
+
+    useEffect(()=> {
+        post('/auth/validate-signature', {})
```

Comment:
So here I'm supposed to access the session storage, and pass the security context to /validate-signature, but now I just pass empty object